apps/openmw/mwrender/animation.cpp

```diff
@@ -1789,9 +1789,12 @@ namespace MWRender
         }
         else
         {
-            effect += 3;
-            float radius = effect * 66.f;
-            float linearAttenuation = 0.5f / effect;
+            // TODO: use global attenuation settings
+
+            // 1 pt of Light effect magnitude corresponds to 1 foot of light source radius, which is about 21.33 game units,
+            // but Morrowind uses imprecise value of foot for magic effects.
+            float radius = effect * 22.f;
```

Comment:
I'd prefer to use UnitsPerFoot here (or ceil(UnitsPerFoot), if we really need 22 here).
The same thing with comment - there is no need to write foot size in units here.